app/src/main/kotlin/com/strv/archdemo/ui/counter/CounterFragment.kt

```diff
@@ -0,0 +1,33 @@
+package com.strv.archdemo.ui.counter
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import androidx.lifecycle.ViewModelProvider
+import com.strv.archdemo.databinding.FragmentCounterBinding
+import com.strv.archdemo.ui.main.MainViewModel
+
+class CounterFragment : Fragment() {
```

Comment:
What about using BaseFragment? 
With some magic for binding and viewmodel that we use on all projects? Too complicated?